packages/theme/src/Theme/Provider.tsx

```diff
@@ -10,24 +10,28 @@ export interface MaskThemeProvider extends React.PropsWithChildren<{}> {
     useTheme(): Theme
     useMaskIconPalette(theme: Theme): MaskIconPalette
     CustomSnackbarOffsetY?: number
+    snackbarType?: SnackbarType
 }
 export function MaskThemeProvider(props: MaskThemeProvider) {
-    const { children, useTheme, useMaskIconPalette, CustomSnackbarOffsetY } = props
+    const { children, useTheme, useMaskIconPalette, CustomSnackbarOffsetY, snackbarType = 'normal' } = props
     const theme = useTheme()
     const MaskIconPalette = useMaskIconPalette(theme)
 
     return compose(
         children,
         (jsx) => <MaskIconPaletteContext.Provider value={MaskIconPalette}>{jsx}</MaskIconPaletteContext.Provider>,
         (jsx) => <ThemeProvider theme={theme} children={jsx} />,
-        (jsx) => (
-            <CustomSnackbarProvider
-                disableWindowBlurListener={false}
-                anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
-                children={jsx}
-                offsetY={CustomSnackbarOffsetY}
-            />
-        ),
+        (jsx) =>
+            snackbarType === SnackbarType.POPUP ? (
```

Comment:
This change does not look right. Why an ad-hoc prop only for Popup? It should be moved out from MaskThemeProvider.